spec/protocolHandler.spec.ts

```diff
@@ -0,0 +1,75 @@
+jest.mock('../src/app/window-actions', () => {
+    return {
+        activate: jest.fn(),
+    };
+});
+
+jest.mock('../src/common/utils', () => {
+    return {
+        getCommandLineArgs: jest.fn(() => 'symphony://?userId=22222'),
+    };
+});
+
+jest.mock('../src/common/env', () => {
+    return {
+        isWindowsOS: false,
+    };
+});
+
+describe('protocol handler', () => {
+    let protocolHandlerInstance;
+
+    beforeEach(() => {
+        jest.resetModules();
+        const { protocolHandler } = require('../src/app/protocol-handler');
```

Comment:
can we not use `import` here?

Reply:
I am using require as a 'trick' to test private functions in typescript